app/frontend/src/pages/Auth/ForgotPassword.css

```diff
@@ -0,0 +1,56 @@
+.cover{
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: #cfc4b0;
+    opacity: 0.2;
+    }
+#container {
+    background-image: url('./images/background.jpg');
+    background-size: cover;
+    background-position: bottom -12rem right 0rem
+                }
+main{
+    position: relative;
+    width: 100%;
+    padding-left: 7%;
+    padding-right: 7%;
+
+     }
+form{
+     display: grid;
+     grid-template-rows: repeat(10,1fr);
+     padding-left: 7%;
+     padding-right: 7%;
+     text-align: center;
+     border-radius: 2%;
+     z-index: 9999;
+ }
+ h1{
+     grid-row: 2;
+
+ }
+ h2{
+     grid-row: 4 / span 3;
+ }
+ input{
+     grid-row: 7;
+     text-align: center;
+ }
```

Comment:
nit: I think we should reconsider these selections based on purely aesthetical reasons. 

For now, it's OK. I will update these in the relevant branches.